src/App.css

```diff
@@ -2,4 +2,16 @@
 .landing-page {
   background-color: #19435A;
   height: 532px;
+}
+
+.error-message-home {
+  position: fixed;
```

Comment:
I really like the Loading... error message in large font on middle of screen! Nicely done.